proto/o5/dante/v1/service/service.proto

```diff
@@ -1,13 +1,13 @@
 syntax = "proto3";
 
-import "o5/dante/v1/dead_message.proto";
-import "o5/auth/v1/actor.proto";
-import "google/protobuf/timestamp.proto";
+import "buf/validate/validate.proto";
 import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
 import "google/protobuf/wrappers.proto";
-import "buf/validate/validate.proto";
+import "o5/auth/v1/actor.proto";
+import "o5/dante/v1/dead_message.proto";
 
-package o5.dante.v1;
+package o5.dante.v1.service;
```

Comment:
@joshuaslate I don't know if this affects your generation or not, but calling it out incase it does. This brings dante in further line with our other stuff.